package.json

```diff
@@ -20,17 +20,18 @@
     "build-release": "./bin/build-release.sh",
     "publish-release": "./bin/publish-release.sh",
     "pre-release": "./bin/pre-release.sh",
-    "predev": "npm run build --workspace govuk-frontend",
+    "predev": "npm run build",
     "dev": "concurrently \"npm run dev --workspace govuk-frontend-review\" \"npm run dev --workspace govuk-frontend\" --kill-others --names \"app,pkg\" --prefix-colors \"red.dim,blue.dim\"",
+    "build": "npm run build --workspace govuk-frontend --workspace govuk-frontend-review",
```

Comment:
Our default build is to prepare the review app for launch

1. Build `govuk-frontend` package
2. Build `govuk-frontend-review` assets

But notice that the webpack example isn't included? For [GitHub Actions **Tests**](https://github.com/alphagov/govuk-frontend/actions/workflows/tests.yml) we instead run:

https://github.com/alphagov/govuk-frontend/blob/01169199015a28fe9779bb814db36bc6f3bb1139/.github/workflows/actions/build/action.yml#L23

By adding a new `npm run build` we can run it before key scripts

E.g. Ensuring we have the `govuk-frontend` package built before we import it

Reply:
@romaricpascal Ah we can't I'm afraid, review app relies on `govuk-frontend/dist/**` things

Any thoughts on using `npm run build` (default) versus `npm run build:package` (named)?

Reply:
Let's narrow it down when we add the stats package (also needs pre-building) in:

* https://github.com/alphagov/govuk-frontend/pull/3681